### ISSUE TITLE
DraftOrder.ShippingLine needs updating

### ISSUE THREAD

**Issue by JoshuaBee** [nozzlegear/ShopifySharp#523]
Currently the Order and DraftOrder class use the same class for their shipping lines, but according to the APIs they use different fields.

[Order API - shipping_lines](https://shopify.dev/docs/admin-api/rest/reference/orders/order?api[version]=2020-04#shipping-lines-property-2020-04)
[DraftOrder API - shipping_lines](https://shopify.dev/docs/admin-api/rest/reference/orders/draftorder?api[version]=2020-04#shipping-line-property-2020-04)

I think this can be resolved by creating a new entity called **DraftShippingLine**, which would look something like this:

```
namespace ShopifySharp
{
  public class DraftShippingLine
  {
    [JsonProperty("custom")]
    public string Custom { get; set; }

    [JsonProperty("handle")]
    public string Handle { get; set; }

    [JsonProperty("title")]
    public string Title { get; set; }

    [JsonProperty("price")]
    public Decimal? Price { get; set; }
  }
}
```

After that we just need to update DraftOrder.ShippingLine to reference DraftShippingLine.

I know it is a small change, but it would make a big difference for my work and helps keep ShopifySharp and the Shopify APIs in line.

**Comment by nozzlegear** [comment]
This has been implemented in 5.3.0 on nuget!

**Comment by JoshuaBee** [comment]
Thank you!
